hr_holidays_hour/tests/test_leave_hours.py

```diff
@@ -0,0 +1,333 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we also should repeat `resource_calendar`'s tests here to be sure what we produced is a drop-in replacement

Reply:
I just copy-pasted the tests from the 'resource' module, but I guess there is an elegant way for repeating those tests avoiding the copy-paste...

Reply:
just import the test class in your test file. Unittest just runs all classes starting with `Test` within the module, also if you just import it